### ISSUE TITLE
package add vim tries to install iceweasel, and fails

### ISSUE THREAD

**Issue by richard-delorenzi** [subuser-security/subuser#349]
I have just installed subuser, and followed the manual. 

I did

    subuser subuser add vim vim@default

and after sometime I get an error:

```
…
Step 9/10 : RUN apt-get install -yqq iceweasel icedove emacs git irssi keepassx libreoffice pandoc vim

 ---> Running in 07ac34078cc6
E: Package 'iceweasel' has no installation candidate
E: Package 'icedove' has no installation candidate  
```
I have got this for all images that I have tried (vim, vlc, wesnoth).

I was going to have a look at the package description to see if I can fix it. However I don't know where to find them. May be it is because `iceweasel` and `icedove` are no longer part of the base image (Debian). I have considered changing to `firefox`, but why are all these other things being installed?

----
Host system: Debian 10 (Buster).

**Comment by richard-delorenzi** [comment]
Partial answer: to find the source of the package descriptions, type

`subuser list repositories --long`

**Comment by richard-delorenzi** [comment]
The problem seems to be that docker-hubs `debian:latest` is not `debian:buster`. And Buster does not have iceweasel or icedove (they sorted out the naming problem, and now use the name filefox etc.

**Comment by richard-delorenzi** [comment]
I have now forked the repo (https://github.com/richard-delorenzi/subuser-default-repository) and make a local clone. The change is not working (do I need to remove then add the repo), so I tried to `subuser repository remove my1`, but I get error `Cannot remove repository my1. Repository is in use. Subusers or installed images exist which rely on this repository.` I then spent half an hour trying to figure out how to remove the images, but still have no idea how.
